src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,18 @@ const createCustomerValidator = () => {
     });
 };
 
+const findCustomerValidator = () => {
+    return celebrate({
+        [Segments.PARAMS]: Joi.object().keys({
+            uuid: Joi
+                .string()
+                .length(36)
```

Comment:
A string aceita qualquer valor (letras, caracteres especiais, barras de espaço...).
Poderia ter restringido o valor para aceitar apenas UUID (universally unique identifier).

```suggestion
                .guid({ version: [ 'uuidv4' ] })
```

Link para referência documentação: https://joi.dev/api/?v=17.6.0#stringguid---aliases-uuid